Velaptor/Observables/ObservableData/ShutDownData.cs

```diff
@@ -0,0 +1,20 @@
+﻿// <copyright file="ShutDownData.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace Velaptor.Observables.ObservableData
+{
+    // ReSharper disable RedundantNameQualifier
+    using System.Diagnostics.CodeAnalysis;
+    using Velaptor.Observables.Core;
+
+    // ReSharper restore RedundantNameQualifier
+
+    /// <summary>
+    /// Holds da ta for the <see cref="IReactor{T}"/>.
```

Comment:
Fix spelling mistake

```suggestion
    /// Holds data for the <see cref="IReactor{T}"/>.
```